docs/organization/dynamic-sampling/index.mdx

```diff
@@ -1,18 +1,54 @@
 ---
-title: Retention Priorities
-sidebar_order: 130
-description: "Learn how Sentry determines retention priorities and how to update them."
+title: Dynamic Sampling
+sidebar_order: 30
+description: "Learn how to make your billing quota go farther with Sentry's Dynamic Sampling."
 ---
 
+## Overview
+
+High-volume projects that generate many events can quickly consume a large portion of your billing quota, leaving less room for other projects and decreasing visibility across them.
+
+With high-volume projects, you'll often see many similar events, which can be redundant and less helpful for debugging. But with Dynamic Sampling, you can maximize your quota by adjusting sample rates based on each project’s event volume. The sample rate is a percentage from 0% (no events passed) to 100% (all events passed), that can be tailored to each project’s needs.
+
+When you use Dynamic Sampling, projects in your organization are sampled based on their recent event volumes. High-volume projects are sampled at lower rates while lower-volume projects are sampled at higher rates or not sampled at all, based on your recent event data. This makes it possible for each project to retain a meaningful set of key events, improving visibility into all projects regardless of their size.
```

Comment:
- For someone not too familiar with the matter, does "event volume" make sense in this context? For me knowing the internals this sounds wrong (we don't classify spans as events) but maybe for a user this is fine. 
- "not sampled" means that everything is dropped. I think you mean to say "sampled at higher rates up to 100%". Also, in this sentence, we don't sample based on event data, but based on event volume.